packages/itwin/grouping-mapping-widget/src/components/Mappings/Extraction/ExtractionStates/SucceededExtractionState.tsx

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { SvgStatusSuccess } from "@itwin/itwinui-icons-color-react";
+import { ANIMATION_DELAY, ANIMATION_DURATION } from "../../../Constants";
+
+interface ExtractionStateProps {
```

Comment:
Both Failed and Succeeded extraction states use the same interface, but it is duplicated. The only difference between these components is `title` and the `SvgStatus*` node.

Reply:
What you could do is keep the `Failed` and `Succeeded` extraction state components, but move the common things to a new component `TerminalExtractionState` that would be used by `Failed` and `Succeeded` extraction states, passing in the `title` and `SvgStatus*` as props.

Reply:
I have made this change.